Homework3/DataAccess.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace start
+{
+    public static class DataAccess
```

Comment:
Will the reader know what data access is for?

Reply:
Renamed this class to UserFileLogic.